rotkehlchen/chain/ethereum/utils.py

```diff
@@ -166,3 +174,38 @@ def should_update_protocol_cache(cache_key: GeneralCacheType, *args: str) -> boo
             key_parts=[cache_key, *args],
         )
     return ts_now() - last_update_ts >= ETH_PROTOCOLS_CACHE_REFRESH
+
+
+def try_download_ens_avatar(
+        eth_inquirer: 'EthereumInquirer',
+        avatars_dir: Path,
+        ens_name: str,
+) -> None:
+    """
+    Handles ens avatar downloading.
+    1. Checks whether given ens name has avatar set
+    2. If it does -- downloads the avatar and saves it in `avatars_dir`
+    3. Updates last avatar check timestamp for the given ens name
+
+    May raise:
+    - RemoteError if failed to query chain
+    - RequestsError if got avatar url, but failed to download it
```

Comment:
What error is this?

Reply:
`RequestsError` is a general error coming from `requests` module. Expected it to be potentially raised here if there is an error with downloading an avatar, but now added handling for it inside this function, so will remove it from the docstring.